fabric-fluid-api-v1/src/main/resources/fabric.mod.json

```
@@ -0,0 +1,34 @@
+{
+  "schemaVersion": 1,
+  "id": "fabric-fluid-api-v1",
+  "name": "Fabric Fluid API (v1)",
+  "version": "${version}",
+  "license": "Apache-2.0",
+  "icon": "assets/fabric-fluid-api-v1/icon.png",
+  "contact": {
+    "homepage": "https://fabricmc.net",
+    "irc": "irc://irc.esper.net:6667/fabric",
+    "issues": "https://github.com/FabricMC/fabric/issues",
+    "sources": "https://github.com/FabricMC/fabric"
+  },
+  "authors": [
+    "FabricMC"
+  ],
+  "depends": {
+    "fabricloader": ">=0.4.0",
+    "fabric-api-base": "*"
+  },
+  "recommends": {
+    "fabric-rendering-fluids-v1": "*"
+  },
```

```suggestion
  "suggests": {
    "fabric-rendering-fluids-v1": "*"
  },
```
I don't think there's a need for warnings if rendering-fluids is absent, that'll most likely only happen in cases where it's intended